### PR TITLE
https://github.com/DANS-KNAW/easy-deposit/issues/3

### PR DESCRIPTION
**Pull request by janvanmansum**
Changed the build to create a tar.gz that can be used as a binary
installation package.

@GeorgiKhomeriki : please, have a look at this.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,8 @@ target/
 data/
 temp/
 
+# Eclipse
```
Comment (janvanmansum) [inline .gitignore:19]: Added some things to ignore, for working in Eclipse.

File: pom.xml [hunk 2]
```diff
@@ -1,23 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>nl.knaw.dans.api</groupId>
+   <parent>
```
Comment (janvanmansum) [inline pom.xml:5]: Added dans-prototype as parent, to inherit installation package building. I will add dans-prototype later to Github, so we don't depend on things that have not been published.

File: pom.xml [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
-  <version>1.0-SNAPSHOT</version>
-  <name>EASY Deposit - SWORD V2.0 deposit service</name>
-  <url>http://maven.apache.org</url>
+  <version>1.x-SNAPSHOT</version>
+  <name>EASY Deposit Service</name>
+  <description>SWORD v2 based dataset deposit service</description>
+  <url>https://github.com/DANS-KNAW/easy-deposit</url>
   <dependencies>
     <dependency>
-      <groupId>junit</groupId>
```
Comment (janvanmansum) [inline pom.xml:12]: We should add unit-tests, but not using junit, and certainly not such an old version :)

File: pom.xml [hunk 4]
```diff
@@ -1,23 +1,25 @@
...
-      <artifactId>junit</artifactId>
-      <version>3.8.1</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
       <groupId>javax.servlet</groupId>
       <artifactId>servlet-api</artifactId>
       <version>2.5</version>
+      <scope>provided</scope>
```
Comment (janvanmansum) [inline pom.xml:22]: Should be provided by the container. Actually, I think we are using a higer version. Something to look into.

File: pom.xml [hunk 5]
```diff
@@ -57,12 +58,22 @@
...
+        <scope>compile</scope>
+    </dependency>
   </dependencies>
   <build>
     <sourceDirectory>src/main/scala</sourceDirectory>
-    <finalName>sword</finalName>
+    <finalName>easy-deposit</finalName>
     <plugins>
       <plugin>
+          <artifactId>maven-assembly-plugin</artifactId>
```
Comment (janvanmansum) [inline pom.xml:74]: Configuration of the plug-in is inhertited from dans-prototype.

File: src/main/assembly/bin.xml [hunk 6]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (janvanmansum) [inline src/main/assembly/bin.xml:1]: Description of the tar.gz to build.

File: src/main/assembly/dist/bin/easy-deposit.xml [hunk 7]
```diff
@@ -0,0 +1 @@
+<Context docBase="/opt/easy-deposit/bin/easy-deposit.war" debug="0" crossContext="true" />
```
Comment (janvanmansum) [inline src/main/assembly/dist/bin/easy-deposit.xml:1]: Deployment descriptor for tomcat.

File: sword.properties [hunk 8]
```diff
@@ -1,5 +0,0 @@
-data-dir:./data
```
Comment (janvanmansum) [inline sword.properties:1]: Changed name to application.properties.

**Comment by gkhom** [comment]
@janvanmansum Looks good to me!

**Pull request merged**